### PR TITLE
Feature: Added A Pull Request Template

### PR DESCRIPTION
**Pull request by SaarthakMaini**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file ?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed ?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### New Feature Submission:

- [x] Does your submission pass the tests? 
- [x] Have you lint your code locally prior to submission?

### Changes To Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Your pull request MUST NOT target the `main` branch on this repository. You probably want to target `staging` instead.

    b. Give a descriptive title to your PR.

    c. Describe your changes.

    d. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Comment by SaarthakMaini** [comment]
Reference Issue: #362

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+
+### New Feature Submission:
+
+- [ ] Does your submission pass the tests? 
+- [ ] Have you lint your code locally prior to submission?
```
Comment (picklelo) [inline .github/pull_request_template.md:21]: Nit: "Have you linted ..."

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+### All Submissions:
+
+- [ ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file ?
```
Comment (picklelo) [inline .github/pull_request_template.md:3]: Nit: remove space before "?"

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed ?
```
Comment (picklelo) [inline .github/pull_request_template.md:4]: Here also

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+### Changes To Core Features:
+
+- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
+- [ ] Have you written new tests for your core changes, as applicable?
+- [ ] Have you successfully ran tests with your changes locally?
+
+### **After** these steps, you're ready to open a pull request.
+
+    a. Your pull request MUST NOT target the `main` branch on this repository. You probably want to target `staging` instead.
```
Comment (picklelo) [inline .github/pull_request_template.md:31]: I think this doesn't apply for us since we do development on the `main` branch

**Review by picklelo — CHANGES_REQUESTED** [review]
Thanks, this will be super useful!

**Pull request closed without merging**